.dockerignore

```
@@ -1 +1,7 @@
-node_modules
+**/dist
+**/node_modules
+.changeset
+.github
+.turbo
+.infra
+out
```

Adding `docker` dir would prevent `base` layer reproduction if any docker config has been edited. do you agree?
```suggestion
out
docker
```

---

Nice catch